main.cpp

```diff
@@ -115,13 +115,13 @@ batteryCharger.chargingStart();
 batteryCharger.chargingStop();
 
 //  8)
-//  Noun:       fridge
-//  action 1:   fridge checks its door status
-fridge.checkDoor();
-//  action 2:   fridge plays warning sound for unclosed door
-fridge.playDoorWarning();
-//  action 3:   fridge extracts temperture information
-fridge.getTemperture();
+//  Noun:       doorbell camera
+//  action 1:   the doorbell camera plays a chime
+doorbellCamera.playChime();
+//  action 2:   the doorbell camera starts a recording
+doorbellCamera.recordingStart();
+//  action 3:   the doorbell camera finishes a recording
+doorbellCamera.recordingStop();

```

Comment:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise